deno/wipe_appwrite_bucket/src/mod.ts

```diff
@@ -0,0 +1,54 @@
+import { sdk } from "./deps.ts";
+
+export default async function(req: any, res: any) {
+  if(
+    !req.variables['APPWRITE_FUNCTION_ENDPOINT'] || 
+    !req.variables['APPWRITE_FUNCTION_PROJECT_ID'] || 
+    !req.variables['APPWRITE_FUNCTION_API_KEY'] ||
+    !req.payload
+  ) {
+    return res.json({
+      success: false,
+      message: "Missing required environment variables or payload.",
+    });
+  }
+  let bucketId = '';
+  try {
+    const payload = JSON.parse(req.payload);
+    bucketId = payload.bucketId;
+  } catch(_err) {
+    return res.json({
+      success: false,
+      message: "Payload is invalid.",
+    });
+  }
+  const client = new sdk.Client();
+  const storage = new sdk.Storage(client);
+  
+  client
+    .setEndpoint(req.variables['APPWRITE_FUNCTION_ENDPOINT'])
+    .setProject(req.variables['APPWRITE_FUNCTION_PROJECT_ID'])
+    .setKey(req.variables['APPWRITE_FUNCTION_API_KEY'])
+  ;
+
+  try {
+    const limit = 25;
+    let lastCursor = '';
+    let listFiles = await storage.listFiles(bucketId, [sdk.Query.limit(limit)]);
+    do {
+      lastCursor = listFiles.files[listFiles.files.length - 1].$id;
```

Comment:
cursor document may cause a problem too especially if the cursor is deleted before the next listFiles() call.

Reply:
That's why I save them before.
What's your suggestion then?